cli.js

```diff
@@ -30,15 +33,13 @@ if (!cli.input[0]) {
   process.exit(1);
 }
 
-psi.output(cli.input[0], cli.flags, function (err, res) {
-  if (err) {
-    if (err.noStack) {
-      console.error(err.message);
-      process.exit(1);
-    } else {
-      throw err;
-    }
-  }
-
+psi.output(cli.input[0], cli.flags).then(function () {
   process.exit(0);
+}, function (err) {
+  if (err.noStack) {
+    console.error(err.message);
+    process.exit(1);
+  } else {
+    throw err;
+  }
```

Comment:
Because of https://github.com/addyosmani/psi/pull/61/files#r43915908, you can remove this catch function.